src/style.css

```diff
@@ -0,0 +1,19 @@
+.block {
+  height: 100px;
+}
+
+.red {
+  background-color: red;
+}
+
+.green {
+  background-color: green;
+}
+
+.blue {
+  background-color: blue;
+}
```

Comment:
It is ok for this task(leave it as is), but usually using style in classname is a bad idea